core/foundation/src/TClassEdit.cxx

```diff
@@ -832,6 +835,9 @@ void TClassEdit::GetNormalizedName(std::string &norm_name, std::string_view name
 {
    norm_name = std::string(name); // NOTE: Is that the shortest version?
 
+   if (name.empty())
+      return;
+
```

Comment:
```suggestion
   if (name.empty())
      return;

   norm_name = std::string(name); // NOTE: Is that the shortest version?

```

Reply:
That would leave `norm_name` with whatever it had before:
```
std::string norm_name{"TObject"};
TClassEdit::NormalizeName(norm_name, "");
/// Now `norm_name` is "TObject"
```
Why would we want that?

Reply:
right, I 'forgot' (did not see from the web interface) that norm_name was the output parameter.  Still, this alternative may (or may not) be 'faster' (depending how smart the optimizer is).
```suggestion
   if (name.empty()) {
      norm_name.clear();
      return;
   }
   
   norm_name = std::string(name); // NOTE: Is that the shortest version?
```